### ISSUE TITLE
trying to verify the sketch error setup wifi not declared in this scope

### ISSUE THREAD

**Issue by MuppetMikey** [tsulis/Duino-Coin-Monitor-OLED-Display#1]
compiling the sketch abd verifying it, i am getting a setup wifi was not declared in this scope error.. how do i fix it?

**Comment by tsulis** [comment]
@MuppetMikey can you paste log error from your IDE?

**Comment by MuppetMikey** [comment]
ok i may be doing this wrong.. i notice you have 2 .ino files the monitor
one, and a common one... how do i load this onto a esp32, and make it work
right do i add the common ino file somehow?

**Comment by MuppetMikey** [comment]
how do i load both files to work is what i am asking..

**Comment by tsulis** [comment]
@MuppetMikey
easy just download those two files and place it in same folder and open "Duino_Coin_OLED.ino". that's it.

or

you can download, everything will be there.
<img width="1122" alt="Screen Shot 2022-09-07 at 11 27 42" src="https://user-images.githubusercontent.com/1165130/188789396-8789e295-7f6f-47d2-8498-5b481ed47935.png">

**Comment by MuppetMikey** [comment]
Ok.. so here is where I am at.. got the code loaded with out errors in the
sketch.. uploaded too an 8266.. itnconnects too wifi and ide serial monitor
shows the format of the stats but all show as 0.0.. almost as if it's not
getting the server information no user name or real stats.. any ideas.. I
could send you what I have going code wise but there is no errors to
show..so that might not help.. or maybe it could..

**Comment by tsulis** [comment]
@MuppetMikey

try this url on your browser, replace **YOUR_DUINO_USERNAME** with your actual duino account username :
https://server.duinocoin.com/v2/users/YOUR_DUINO_USERNAME

it should show your account if your account is exist, somehing like this :
<img width="1658" alt="Screen Shot 2022-09-11 at 11 45 17" src="https://user-images.githubusercontent.com/1165130/189512923-45696174-29c2-459c-be3c-3e7d50ba543e.png">

**Comment by MuppetMikey** [comment]
yes i have done that and it works.. the issue i am having is with the
sketch not seeming to get the same information with the code i am running..
i have attached 2 images of the serial monitor that shows the result of
adding the user name into the sketch manually

**Comment by MuppetMikey** [comment]
is it possible that the 8266 isnt really connected to the internet?? that
would explain the blank feilds.. how to i check to see if the code is
connecting to internet?

**Comment by tsulis** [comment]
@MuppetMikey ,

once you successfully program your board with your setting, you can open serial monitor on your IDE, it will show you information

**Comment by MuppetMikey** [comment]
ok got it in the same folder and loaded with out errors.. but looking there
is nothing on screen, and getting       deserializjson () failed.. no
memory  .. this is on the serial monitor
of ide.. nothing on screen showing

**Comment by tsulis** [comment]
@MuppetMikey,

on that issue you can increase "**DynamicJsonDocument doc (8000)**;"  more than 8000, for example 10000. but do remember, do not over increase this value, this buffer uses actual board memory.

<img width="753" alt="Screen Shot 2022-10-11 at 14 38 41" src="https://user-images.githubusercontent.com/1165130/195025800-f2f612fe-b114-4e34-866f-7c951324bf42.png">

**Comment by dunk8888** [comment]
Ime having same problem no input

**Comment by dunk8888** [comment]
Shall i up my flash size from 4 to 8 mb?
